src/Resources/components/ItemUnits/Form/index.js

```diff
@@ -97,7 +97,7 @@ class ItemUnitForm extends Component {
             },
             () => {
               notifyError(
-                'Something occurred while saving Unit of measure of measure, please try again!'
+                'Something occurred while saving unit of measure of measure, please try again!'
```

Comment:
You forgot this part here

Reply:
Also check for words repetition "of measure of measure"